packages/@aws-cdk/applet-js/bin/cdk-applet-js.ts

```diff
@@ -34,7 +34,7 @@ async function main() {
     throw new Error(`${appletFile}: must have an 'applets' key`);
   }
 
-  const searchDir = path.dirname(appletFile);
+  const searchDir = path.dirname(path.resolve(process.cwd(), appletFile));
```

Comment:
No need to include `process.cwd()` in the resolve call. A resolve with a single argument will do.